beacon_node/lighthouse_network/src/rpc/protocol.rs

```diff
@@ -399,6 +399,9 @@ impl SupportedProtocol {
             supported.extend_from_slice(&[
                 ProtocolId::new(SupportedProtocol::BlobsByRootV1, Encoding::SSZSnappy),
                 ProtocolId::new(SupportedProtocol::BlobsByRangeV1, Encoding::SSZSnappy),
+                // TODO(das): add to PeerDAS fork
+                ProtocolId::new(SupportedProtocol::DataColumnsByRootV1, Encoding::SSZSnappy),
+                ProtocolId::new(SupportedProtocol::DataColumnsByRangeV1, Encoding::SSZSnappy),
```

Comment:
Should delete this. It looks like it effects Deneb when it shouldn't.

Reply:
Done